components/script/script_thread.rs

```diff
@@ -1416,7 +1416,7 @@ impl ScriptThread {
             replace_surrogates: opts.debug.replace_surrogates,
             user_agent,
             player_context: state.player_context,
-            event_loop_waker: state.event_loop_waker,
```

Comment:
Oh, one question here. Is this ever not-None?

Reply:
It shouldn't be not-None. In ScriptThreadFactory when we create a ScriptThread the event_loop_waker is already None(in pipeline.rs `STF::create()`). And I also don't see this function being called elsewhere.

Reply:
Could you also remove the `event_loop_waker` member from `ScriptThread` in that case as well?

Reply:
Removed in [339aed9](https://github.com/servo/servo/pull/30101/commits/339aed92cb6fa15a59bae9d47e6c32d53f95bc06), thanks.